### PR TITLE
chore: Moves the images folder to the assets folder

### PR DESCRIPTION
**Pull request by michael-s-molina**
### SUMMARY
Moves the `images` folder to the `assets` folder.

This work is part of [SIP-61](https://github.com/apache/superset/issues/13632)

@rusackas @pkdotson please include anyone that needs to be aware of this. I tested the Storybook, development, and production versions and it seems ok. Please let me know if I forgot any dependency.

PS: Don't panic about the 317 files changed 😆, most are just file renames.

### TEST PLAN
- Create an ephemeral environment and test the application
- Test the Storybook
- Test the build process 

### ADDITIONAL INFORMATION
- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Review by geido — CHANGES_REQUESTED** [review]
Hey @michael-s-molina we need to make sure that the path is changed for the dynamic import of the Icons component as well. I don't see that path change included here

**Comment by michael-s-molina** [comment]
> Hey @michael-s-molina we need to make sure that the path is changed for the dynamic import of the Icons component as well. I don't see that path change included here

@geido This one?

Screen Shot 2021-05-06 at 2 03 38 PM

**Comment by geido** [comment]
> > Hey @michael-s-molina we need to make sure that the path is changed for the dynamic import of the Icons component as well. I don't see that path change included here
> 
> @geido This one?
> 
> Screen Shot 2021-05-06 at 2 03 38 PM
> 

Is this in this PR? I have looked 3 times. I can't find it.

**Comment by geido** [comment]
> > Is this in this PR? I have looked 3 times. I can't find it.
> 
> @geido https://github.com/apache/superset/pull/14429/files#diff-4ee0a8eefdf7b42a39b21a7183b6559a4909da71533312d9eba61c569234862f

That does not bring me to that change. I am from the Github app, maybe it is messed up due to the big amount of changes. But if you confirm it is there, then we are good @michael-s-molina

File: superset-frontend/src/assets/staticPages/404.html [hunk 1]
```diff
@@ -94,14 +94,14 @@
       <section>
         <h1>Page not found</h1>
         <p>
-          Sorry, we cannot find t he page you are looking for. You may have
+          Sorry, we cannot find the page you are looking for. You may have
```
Comment (rusackas) [inline superset-frontend/src/assets/staticPages/404.html:97]: Nice catch :)

**Review by etr2460 — COMMENTED** [review]
it looks like a lot of the images in the directory are only used for documentation or on the docs website. can you verify before merging that they don't get deployed to the docker image, since they aren't needed to actually run the server?

**Review by geido — APPROVED** [review]
LGTM! Thanks!

**Comment by villebro** [comment]
> it looks like a lot of the images in the directory are only used for documentation or on the docs website. can you verify before merging that they don't get deployed to the docker image, since they aren't needed to actually run the server?

I can confirm that I was not able to find the doc/website images in the docker image. So should be good to go 👍

**Review by villebro — APPROVED** [review]
LGTM!

**Pull request merged**